lib/src/bridge/device_connection_mixin.dart

```diff
@@ -39,16 +39,16 @@ mixin DeviceConnectionMixin on FlutterBLE {
           return PeripheralConnectionState.disconnecting;
         default:
           throw FormatException(
-              "Unrecognized value of device connection state. Value: $connectionStateString");
+              'Unrecognized value of device connection state. Value: $connectionStateString');
       }
     });
 
-    _methodChannel.invokeMethod(
+    unawaited(_methodChannel.invokeMethod(
```

Comment:
@mikolak Especially your attention should be here. I am not sure if it was "unawaited" intentionally and I lack a bit to be sure now.

Reply:
IIRC the values might start coming before the future returns. We were trying to make the call non-blocking, so that first values could come already. This was just as we started working on the library and we wanted to avoid stream controllers by using yield*. 

This can be improved by declaring a `StreamController` and passing the invoking of the method to its `onListen` instead, same construction as in monitoring characteristics.